Description: Upstream changes introduced in version 0.0.1+svn20110409.0930-6yavdr1~oneiric
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 eventlircd (0.0.1+svn20110409.0930-6yavdr1~oneiric) oneiric; urgency=low
 .
   * removed last change
   * removed -flto from linker flags, breaks build on oneiric
 .
 The person named in the Author field signed this changelog entry.
Author: Gerald Dachs <gda@dachsweb.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- eventlircd-0.0.1+svn20110409.0930.orig/configure.ac
+++ eventlircd-0.0.1+svn20110409.0930/configure.ac
@@ -119,7 +119,6 @@ if ! echo " ${CC} ${CFLAGS} " | ${GREP}
     AX_C_CHECK_FLAG([-O2],[],[],[CFLAGS="$CFLAGS -O2"],[])
 fi
 AX_C_CHECK_FLAG([-pedantic],[],[],[CFLAGS="$CFLAGS -pedantic"],[])
-AX_C_CHECK_FLAG([-flto],[],[],[CFLAGS="$CFLAGS -flto"],[])
 AX_C_CHECK_FLAG([-fvisibility=hidden],[],[],[CFLAGS="$CFLAGS -fvisibility=hidden"],[])
 AX_C_CHECK_FLAG([-Wall],[],[],[CFLAGS="$CFLAGS -Wall"],[])
 AX_C_CHECK_FLAG([-Wextra],[],[],[CFLAGS="$CFLAGS -Wextra"],[])
@@ -134,7 +133,6 @@ AX_C_CHECK_FLAG([-Wmissing-include-dirs]
 AX_C_CHECK_FLAG([-Wshadow],[],[],[CFLAGS="$CFLAGS "],[-Wshadow])
 AX_C_CHECK_FLAG([-Wstrict-overflow=5],[],[],[CFLAGS="$CFLAGS -Wstrict-overflow=5"],[])
 AX_C_CHECK_FLAG([-Wundef],[],[],[CFLAGS="$CFLAGS -Wundef"],[])
-AX_LD_CHECK_FLAG([-flto],[],[],[LDFLAGS="$LDFLAGS -flto"],[])
 AX_LD_CHECK_FLAG([-fuse-ld=gold],[],[],[LDFLAGS="$LDFLAGS -fuse-ld=gold"],[])
 AX_LD_CHECK_FLAG([-fwhole-program],[],[],[LDFLAGS="$LDFLAGS -fwhole-program"],[])
 AX_LD_CHECK_FLAG([-fvisibility=hidden],[],[],[LDFLAGS="$LDFLAGS -fvisibility=hidden"],[])
--- eventlircd-0.0.1+svn20110409.0930.orig/src/Makefile.am
+++ eventlircd-0.0.1+svn20110409.0930/src/Makefile.am
@@ -2,7 +2,7 @@ sbin_PROGRAMS = eventlircd
 eventlircd_SOURCES = main.c monitor.c monitor.h input.c input.h lircd.c lircd.h
 eventlircd_CFLAGS = $(AM_CFLAGS) $(LIBUDEV_CFLAGS) -DLIRCD_SOCKET=\"$(LIRCD_SOCKET)\" -DEVMAP_DIR=\"$(EVMAP_DIR)\"
 eventlircd_LDFLAGS = $(AM_CFLAGS)
-eventlircd_LDADD = $(LIBUDEV_LIBS) -lrt
+eventlircd_LDADD = $(LIBUDEV_LIBS)
 
 nodist_eventlircd_SOURCES = event_name_to_code.h evkey_code_to_name.h evkey_type.h
 
